src/api/java/com/minecolonies/coremod/colony/permissions/IPermissions.java

```diff
@@ -14,10 +14,10 @@
      * Returns whether the player has the permission for an action.
      *
      * @param player {@link EntityPlayer} player.
-     * @param action {@link com.minecolonies.coremod.colony.permissions.Permissions.Action} action.
+     * @param action {@link Action} action.
      * @return true if has permission, otherwise false.
      */
-    boolean hasPermission(EntityPlayer player, Permissions.Action action);
+    boolean hasPermission(EntityPlayer player, Action action);
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready